[2024-01-17T13:58:43.234+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: sql_example.create_update_dim_customer manual__2024-01-17T13:58:37.009006+00:00 [queued]>
[2024-01-17T13:58:43.252+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: sql_example.create_update_dim_customer manual__2024-01-17T13:58:37.009006+00:00 [queued]>
[2024-01-17T13:58:43.253+0000] {taskinstance.py:2171} INFO - Starting attempt 1 of 0
[2024-01-17T13:58:43.280+0000] {taskinstance.py:2192} INFO - Executing <Task(BigQueryExecuteQueryOperator): create_update_dim_customer> on 2024-01-17 13:58:37.009006+00:00
[2024-01-17T13:58:43.335+0000] {standard_task_runner.py:60} INFO - Started process 7574 to run task
[2024-01-17T13:58:43.355+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'sql_example', 'create_update_dim_customer', 'manual__2024-01-17T13:58:37.009006+00:00', '--job-id', '44', '--raw', '--subdir', 'DAGS_FOLDER/data_transform_BQ.py', '--cfg-path', '/tmp/tmpbsc17m_9']
[2024-01-17T13:58:43.385+0000] {standard_task_runner.py:88} INFO - Job 44: Subtask create_update_dim_customer
[2024-01-17T13:58:43.595+0000] {task_command.py:423} INFO - Running <TaskInstance: sql_example.create_update_dim_customer manual__2024-01-17T13:58:37.009006+00:00 [running]> on host fafdf46f827b
[2024-01-17T13:58:43.780+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='phungquan' AIRFLOW_CTX_DAG_ID='sql_example' AIRFLOW_CTX_TASK_ID='create_update_dim_customer' AIRFLOW_CTX_EXECUTION_DATE='2024-01-17T13:58:37.009006+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-01-17T13:58:37.009006+00:00'
[2024-01-17T13:58:43.786+0000] {bigquery.py:1245} INFO - Executing: CREATE OR REPLACE TABLE 
  `OLAP.Dim_Customers` AS
SELECT
  customers.CustomerID,
  customers.City,
  customers.Phone,
  customers.Country
FROM
  `OLTP.Customers` AS customers
[2024-01-17T13:58:43.820+0000] {connection.py:234} WARNING - Connection schemes (type: google_cloud_platform) shall not contain '_' according to RFC3986.
[2024-01-17T13:58:43.826+0000] {base.py:83} INFO - Using connection ID 'google_cloud_default' for task execution.
[2024-01-17T13:58:43.841+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.8/site-packages/***/providers/google/cloud/hooks/bigquery.py:2109 AirflowProviderDeprecationWarning: This method is deprecated. Please use `BigQueryHook.insert_job` method.
[2024-01-17T13:58:43.842+0000] {credentials_provider.py:353} INFO - Getting connection using `google.auth.default()` since no explicit credentials are provided.
[2024-01-17T13:58:43.977+0000] {bigquery.py:1596} INFO - Inserting job ***_1705499923974297_110dc7cdc8723b206594a2ec75de8f20
[2024-01-17T13:58:46.757+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=sql_example, task_id=create_update_dim_customer, execution_date=20240117T135837, start_date=20240117T135843, end_date=20240117T135846
[2024-01-17T13:58:46.868+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-01-17T13:58:46.971+0000] {taskinstance.py:3281} INFO - 0 downstream tasks scheduled from follow-on schedule check
